.eslintcache

```diff
@@ -1 +1 @@
-[{"C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\nutrimed\\src\\index.js":"1","C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\nutrimed\\src\\reportWebVitals.js":"2","C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\nutrimed\\src\\App.js":"3"},{"size":500,"mtime":499162500000,"results":"4","hashOfConfig":"5"},{"size":362,"mtime":499162500000,"results":"6","hashOfConfig":"5"},{"size":528,"mtime":499162500000,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"198e47k",{"filePath":"10","messages":"11","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\nutrimed\\src\\index.js",[],"C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\nutrimed\\src\\reportWebVitals.js",[],"C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\nutrimed\\src\\App.js",[]]
\ No newline at end of file
+[{"C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\src\\index.js":"1","C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\src\\App.js":"2","C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\src\\reportWebVitals.js":"3"},{"size":500,"mtime":499162500000,"results":"4","hashOfConfig":"5"},{"size":770,"mtime":1606900641692,"results":"6","hashOfConfig":"5"},{"size":362,"mtime":499162500000,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"nvpdmo",{"filePath":"10","messages":"11","errorCount":0,"warningCount":1,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\src\\index.js",[],"C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\src\\App.js",["14"],"C:\\Users\\alexandru.ciobotaru\\Desktop\\Lucru\\nutrimed\\src\\reportWebVitals.js",[],{"ruleId":"15","severity":1,"message":"16","line":4,"column":8,"nodeType":"17","messageId":"18","endLine":4,"endColumn":16},"no-unused-vars","'ReactDOM' is defined but never used.","Identifier","unusedVar"]
```

Comment:
this file should not be included in the repo and it should be added in the .gitignore configuration